thrimshim/thrimshim/main.py

```diff
@@ -178,11 +178,11 @@ def get_row(ident):
 	]
 	if response['event_start'] is not None:
 		start = response['event_start']
-		if response['thumbnail_template'] is None:
+		if 'thumbnail_template' not in response or response['thumbnail_template'] is None:
```

Comment:
`'thumbnail_template' not in response` should never happen. we're fetching a database row, we can assume the schema is correct and all columns are present.

Reply:
When I added this it's because it was crashing for not existing. It's *possible* I forgot to destroy the old database before running this (I don't remember now). I'll remove this change assuming that's the case.